Makefile

```diff
@@ -66,7 +66,7 @@ setup-git: ensure-venv setup-git-config
 	@echo "--> Installing git hooks"
 	cd .git/hooks && ln -sf ../../config/hooks/* ./
 	@PYENV_VERSION=$(REQUIRED_PY3_VERSION) python3 -c '' || (echo 'Please run `make setup-pyenv` to install the required Python 3 version.'; exit 1)
-	$(PIP) install "pre-commit==1.18.2"
+	$(PIP) install "pre-commit==1.18.2" "virtualenv==20.0.22"
```

Comment:
isn't .22 the broken version?

Reply:
I haven't bumped it yet. I'm holding my breath to see if 20.0.23 actually works.